entities/Client.php

```diff
@@ -58,13 +62,19 @@ public function create(){
 
     }
     public function readAll(){
-        if(is_admin()&&$_SESSION["organization_id"]==-99){
```

Comment:
This merge has removed the logic of the current organization being -99 from here